### ISSUE TITLE
What is the expected loss range during fine-tuning?

### ISSUE THREAD

**Issue by bruceLogic** [mozi1924/Qwen3-TTS-EasyFinetuning#4]
Hi, thanks for this great project!

I'm fine-tuning Qwen3-TTS using sft_12hz.py and would like to ask about the expected loss values during training.

From the code, the total loss is computed as:

loss = outputs.loss + 0.3 * sub_talker_loss

Where outputs.loss is the CrossEntropyLoss for codec layer 0 (main talker), and sub_talker_loss is the CrossEntropyLoss for codec layers 1–15 (sub-talker / code predictor).

I have a few questions:

What is the typical range of talker_loss and sub_talker_loss after the model converges well?
At what point would you consider the loss "too high" (i.e., something is wrong with training)?
Any reference numbers or training curves from your experiments would be very helpful. Thanks!
